aws-api/src/main/java/com/amplifyframework/api/aws/GsonGraphQLResponseFactory.java

```diff
@@ -55,9 +55,9 @@
     }
 
     @Override
-    public <T> GraphQLResponse<T> buildResponse(GraphQLRequest<T> request, String responseJson, Type typeOfT)
+    public <T> GraphQLResponse<T> buildResponse(GraphQLRequest<T> request, String responseJson)
```

Comment:
This is the main change.  We don't need to pass `request` _and_ `typeOfT` because we can get `typeOfT` via `request.getResponseType()`.